Dockerfile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+# This Dockerfile should be run from the parent directory of the egeria-connector-sas-viya directory
+# ie
+# docker -f ./Dockerfile 
+
+FROM odpi/egeria
```

Comment:
We will want to retrieve the same version of egeria as that we are building.

We should make this version an argument - docker now supports this - see https://www.jeffgeerling.com/blog/2017/use-arg-dockerfile-dynamic-image-specification . Default could be 'latest' in case someone runs it without arg.